sql/src/main/java/io/crate/planner/projection/GroupProjection.java

```diff
@@ -54,6 +56,12 @@ public GroupProjection(List<Symbol> keys, List<Aggregation> values) {
         this.values = values;
     }
 
+    public GroupProjection(List<Symbol> keys, List<Aggregation> values,
```

Comment:
maybe add docs to the constructor and explain that limit can only be used if the Projector receives its input pre-sorted